cvat/apps/annotation/cvat_ddln.py

```diff
@@ -0,0 +1,125 @@
+# Copyright (C) 2020
+
+format_spec = {
+    "name": "DDLN_CSV",
+    "dumpers": [
+        {
+            "display_name": "{name} {format} {version} for images (BB)",
+            "format": "ZIP",
+            "version": "0.1",
+            "handler": "dump_ddln_csv"
+        }
+    ],
+    "loaders": [
+        {
+            "display_name": "{name} {format} {version}",
+            "format": "CSV",
+            "version": "0.1",
+            "handler": "load",
+        }
+    ],
+}
+
+def extractFileParams(filename):
+    directoryName = filename.split("/")[2]
+    csvFilename = filename.split("/")[-1]
+    csvFilename = csvFilename[:-4] + "_y_bb.csv"
+	
+    return directoryName,csvFilename
+
+def writeToCsv(dirname, filename, data):
+    try:
+        if not os.path.exists(dirname):
+            os.mkdir(dirname)
+        with open(dirname + filename, 'w', newline='') as csvfile:
+            csvfile.write(data)            
+    except Exception as e:
+        return e  #for debugging
+            
+    return True
+
+def dump_ddln_csv(file_object, annotations):
+    from cvat.apps.dataset_manager.util import make_zip_archive
+    from tempfile import TemporaryDirectory
+
+    with TemporaryDirectory() as temp_dir:
+        log_file_path = temp_dir + "/" + "export.log"
+        log_data = ""; 
```

Comment:
write to log file as you go, no point in collecting the whole log in a string first